### PR TITLE
UI improvements and bug fixes

### PR DESCRIPTION
**Pull request by MurtazaBagwala07**
Created a better modal layout
Improved layout for Notes ,archived and lables page
Fixed input reset after adding note bug
Fixed broken links in Navbar

### CODE REVIEW

**Review by saifudinbadani — APPROVED** [review]
The changes made are easy to understand, it would be great if you can mention what does div with id='portal' do?

File: memento-notes/src/components/Notecard/Notecard.css [hunk 1]
```diff
@@ -16,4 +16,115 @@
...
+  .note-action-btn {
+    width: 3rem;
+    height: 3rem;
+    border-radius: 50%;
+    background-color: var(--primary-color);;
+    border: 0.1rem solid var(--secondary-color);;
+    font-weight: 900;
+    font-size: 1rem;
+    outline: none;
+    color: var(--secondary-color);;
```
Comment (ravikumar1002) [inline memento-notes/src/components/Notecard/Notecard.css]: remove this extra semicolon

**Review by ravikumar1002 — APPROVED** [review]
naming is good and code is properly formatted
good to merge
one suggestion : you can also add some SS or video for UI look in description box

File: memento-notes/src/components/Notecard/Notecard.css (hunk 1 above)
Comment (MurtazaBagwala07) [reply memento-notes/src/components/Notecard/Notecard.css]: done

**Pull request merged**